tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/bluetooth.py

```diff
@@ -4,55 +4,61 @@
 
 from marionette_driver import expected, By, Wait
 
-from gaiatest.apps.base import PageRegion
+from gaiatest.apps.base import Base
```

Comment:
wait, if so then I'll fail johan's script. okay, i think my workaround is simple enough